content/src/fieldConfig/profile.json

```diff
@@ -101,26 +112,30 @@
       "taxPin": {
         "default": {
           "header": "Tax PIN",
+          "headerContextualInfo": "",
           "description": "What is your `Tax PIN|tax-pin`?",
           "errorTextRequired": "Tax PIN must be 4 digits."
         }
       },
       "businessName": {
         "default": {
           "header": "Business Name",
+          "headerContextualInfo": "",
           "errorTextRequired": "Please enter your business name.",
-          "description": "What will you name your business?"
+          "description": ""
         },
-        "overrides": {
-          "OWNING": {
-            "description": "What is your business' name?"
+        "onboarding" : {
+          "default": {
+            "description": "What will you name your business?"
```

Comment:
Why this change? I didn't think business name ever got asked in onboarding?

Reply:
That is correct and I will fix it